docs/guides/websites/cms/wordpress/best-open-source-content-management-systems/index.md

```diff
@@ -0,0 +1,308 @@
+
+### News and Blog Publishing
+
+Here the focus is on publishing words and possibly audio and video content. You need ways for editors and writers to prepare and publish content. You also need to be able to quickly add and change complex links, and to categorize the content. Another important element is the ability to link your content to social media and network platforms.
+
+[WordPress](/docs/guides/websites/cms/wordpress/) is by far the most well-known of these CMSs, but there are many others.
+
+### Social Media
+
+Developing an online community site is much more complicated than a simple website. Here content is created not only by you, but by your sites' members as well. This means you must give them the tools they need to create and share content.
+
+While social networking sites such as Facebook and Twitter are based on open-source, you can't use their code to make your own Facebook clone. There are many open-source social media CMSs, such as [Elgg](https://elgg.org/), [Mastodon](/docs/guides/install-mastodon-on-ubuntu-2004/), and [Dolphin](https://github.com/boonex/dolphin.pro), but none of them, or services based on them, have gained anything close to the success of the major social networks. Still, if you want to build a social network of your own, these give you the foundation you need.
+
+### Content Management System Types
+
+There are technically two different kinds of CMSs. The first are conventional CMSs, such as the well-known [WordPress](https://wordpress.com/), [Joomla!](https://www.joomla.org/), and [Drupal](https://www.drupal.org/). These help you build both the backend and frontend of your website. The newer headless CMS takes a radically different approach.
+
+A headless CMS doesn't bother with the website's front-end. Instead, its focus is on the back-end content repository, which is used for storing and delivering structured content. This content is then made available for display via a RESTful API, typically using [JSON](https://www.json.org/) or [XML](https://www.w3schools.com/xml/xml_whatis.asp).
+
+The argument for this approach is that traditional CMSs, such as the 20-plus-year-old WordPress, have a monolithic legacy architecture that is difficult to build, often slow, and vulnerable to hackers. Headless CMS, such as [Strapi](https://strapi.io/), [Ghost](https://ghost.org/), [Contentful](https://www.contentful.com/), and [Prismic](https://prismic.io/), use APIs to integrate into a variety of front-end frameworks, which gives developers the freedom to choose their favorite tools, while reducing hosting and development costs.
+
+With all this in mind, the section below includes some of the best open-source CMSs to consider. There is no ‘best’ or ‘worst’ between these tools. Fundamentally, it's a question of the right fit for you.
+
+## The Top Ten Open-Source Content Management Systems
+
+### Bitrix24
```

Comment:
I think we should remove Bitrix24 and find another CMS to use here instead. It's not really a CMS (more of a CRM and project management system) and, like this guide mentions, isn't open source.

Reply:
I was on the fence here @andystevensname what do you think?

Reply:
Maybe highlight Grav. It's a GREAT CMS, open source, and we offer it as a Marketplace App.